client/components/auth/register.js

```diff
@@ -181,7 +171,7 @@ class Register extends Component {
                                     onBlur={this.onBlur}
                                     onFocus={this.onFocus}
                                     type="password" className="validate" />
-                                <label htmlFor="password-confirm">Password Confirmation</label>
+                                <label htmlFor="cpassword">Password Confirmation</label>
```

Comment:
What informed your decision of changing the label from `password-confirm` to `cpassword`? I believe the first name is more descriptive...

Reply:
I noticed I could not use the "_password-confirm_" as the key of objects so I had to change it to a single word.

Reply:
For that you could use camel case like: `passwordConfirm` or `confirmPassword`